src/prefect/tasks/prometheus/pushgateway.py

```diff
@@ -0,0 +1,137 @@
+from enum import Enum
+from prometheus_client import (
+    CollectorRegistry,
+    Gauge,
+    pushadd_to_gateway,
+    push_to_gateway,
+)
+from typing import Dict, List, Optional
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class PushMode(Enum):
```

Comment:
are there more than two push modes?
the `Enum` seems a bit heavy, since users will have to import and configure `PushMode`

if it's just `Push` and `PushAdd`, can we replace this with a boolean flag?

Reply:
Sure, I can. I was thinking as well as having to task, one for each mode as it is a different call as well in the API. What do you think ?

Reply:
I think one task is okay for both, but I'm not as familiar with Prometheus,